.github/workflows/release.yml

```diff
@@ -6,7 +6,17 @@ on:
       - published
 
 jobs:
+  prerelease:
+    if: "github.event.release.prerelease"
+    permissions:
+      id-token: write
+      contents: write
+    uses: SonarSource/gh-action_release/.github/workflows/main.yaml@v5
+    with:
+      publishToTestPyPI: true
+      skipPythonReleasabilityChecks: true
```

Comment:
Is there a reason to skip those checks ? I didn't really find much documentation on that

Reply:
Oh yes sorry I had to dig into the implementation to find that for the moment these checks are not reliable: https://github.com/SonarSource/gh-action_release/blob/a6f57de1b1c23b7cd4f4eceab350ee4aca2cca0d/.github/workflows/main.yaml#L69